modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/utils/IconProcessorUtil.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.orchestrator.utils;
+
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import javax.imageio.ImageIO;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.net.util.Base64;
+
+/**
+ * IconProcessorUtil.
+ */
+public class IconProcessorUtil {
```

Comment:
It looks like the `IconProcessorUtil` isn't involved.

Reply:
The change branch lost part of the code, I will modify it.